Foundation/NSISO8601DateFormatter.swift

```diff
@@ -10,70 +10,111 @@
 import CoreFoundation
```

Comment:
Yeah I agree, I'll remove the NS from this file and the test file as well. 
Thanks for the review